Dockerfile

```diff
@@ -1,93 +1,97 @@
-FROM ubuntu:18.04
-MAINTAINER Quartile Limited <info@quartile.co>
+FROM alpine/git as odoo-code
+LABEL maintainer="Quartile Limited <info@quartile.co>"
 
-RUN apt-get update && apt-get install -y gnupg
+ARG ODOO_SOURCE=odoo/odoo
+ARG ODOO_VERSION=14.0
 
-# Update source repository
-RUN apt-key adv --keyserver keyserver.ubuntu.com --recv-keys B97B0AFCAA1A47F044F244A07FCC7D46ACCC4CF8 \
-    && echo "deb http://apt.postgresql.org/pub/repos/apt/ xenial-pgdg main" > /etc/apt/sources.list.d/pgdg.list \
-    && apt-get update
+WORKDIR /
+RUN git clone --depth 1 https://github.com/$ODOO_SOURCE.git -b $ODOO_VERSION
 
-# Set Environment Variable
-ENV LC_ALL=C.UTF-8
 
-# Install dependencies and tools
-RUN set -x; \
-  apt-get install -yq --no-install-recommends \
-    curl \
-    python3-pip \
-    # Libraries needed to install the pip modules (libpq-dev for pg_config > psycopg2)
-    python3-dev \
-    # to install portable C which is a distant dependency for pysftp 
-    libffi-dev \
-    libxml2-dev \
-    libxslt1-dev \
-    libldap2-dev \
-    libsasl2-dev \
-    libssl-dev \
-    python3-setuptools \
-    build-essential \
-    # For database management
-    postgresql-client-10 \
-    # GeoIP related
-    # geoip-database-contrib \
-    libgeoip-dev \
-    # For getting Odoo code
-    git
-
-# Install Odoo Python dependencies.
-ADD requirements.txt /opt/requirements.txt
-RUN python3 -m pip install --upgrade pip \
-  && pip3 install -r /opt/requirements.txt
-
-# Install LESS
+FROM python:3.6-slim-buster
+LABEL maintainer="Quartile Limited <info@quartile.co>"
+
+ARG WKHTMLTOPDF_VERSION=0.12.5
+ARG WKHTMLTOPDF_CHECKSUM=1140b0ab02aa6e17346af2f14ed0de807376de475ba90e1db3975f112fbd20bb
+ARG ODOO_SOURCE=odoo/odoo
+ARG ODOO_VERSION=14.0
+
+# Generate locale C.UTF-8 for postgres and general locale data
+ENV LANG C.UTF-8
+
+COPY --from=odoo-code odoo /opt/odoo
```

Comment:
As I check the image size again, I am not sure if the multi-stage build is helping here. Better double check.